src/node_modules.cc

```diff
@@ -522,7 +523,7 @@ void BindingData::CreatePerContextProperties(Local<Object> target,
   Realm* realm = Realm::GetCurrent(context);
   realm->AddBindingData<BindingData>(target);
 
-  std::vector<Local<Value>> compile_cache_status_values;
+  LocalVector<Value> compile_cache_status_values(isolate);
   Isolate* isolate = context->GetIsolate();
```

Comment:
CI is failing with `‘isolate’ was not declared in this scope` here. Looks like it just needs reordering:

```suggestion
  Isolate* isolate = context->GetIsolate();
  LocalVector<Value> compile_cache_status_values(isolate);
```

Reply:
Weird. No idea how that happened. I think that maybe ended up being an accidental line move in my editor.